ctru-rs/Cargo.toml

```diff
@@ -1,11 +1,15 @@
 [package]
-authors = ["Rust3DS Org", "Ronald Kinard <furyhunter600@gmail.com>"]
-description = "A safe wrapper around smealum's ctrulib."
-license = "Zlib"
 name = "ctru-rs"
 version = "0.7.1"
+authors = ["Rust3DS Org", "Ronald Kinard <furyhunter600@gmail.com>"]
+description = "A safe wrapper around libctru"
+repository = "https://github.com/rust3ds/ctru-rs"
+keywords = ["3ds", "libctru"]
+categories = ["os", "api-bindings"]
+exclude = ["examples"]
```

Comment:
Yes, people suggest to do so since the crates pulled from `crates.io` end up in the `cargo` cache and aren't easily accessible. It's just meant to keep the archive size small.